Source/Core/Common/ArmFPURoundMode.cpp

```diff
@@ -45,8 +47,25 @@ void SetPrecisionMode(PrecisionMode mode)
 
 void SetSIMDMode(int rounding_mode, bool non_ieee_mode)
 {
-  // Flush-To-Zero (non-IEEE mode: denormal outputs are set to +/- 0)
+  // When AH is disabled, FZ controls flush-to-zero for both inputs and outputs. When AH is enabled,
+  // FZ controls flush-to-zero for outputs, and FIZ controls flush-to-zero for inputs.
   constexpr u32 FZ = 1 << 24;
+  constexpr u32 AH = 1 << 1;
+  constexpr u32 FIZ = 1 << 0;
+  constexpr u32 flush_to_zero_mask = FZ | AH | FIZ;
+
+  // Setting AH = 1, FZ = 1, FIZ = 0 emulates the GC/Wii CPU's "non-IEEE mode". Unfortunately,
+  // the AH and FIZ bits were not added until 2020, so we can't count on setting AH actually
+  // doing anything. But flushing both inputs and outputs seems to cause less problems than
+  // flushing nothing, so let's just set FZ and AH and roll with whatever behavior we get.
```

Comment:
Would this change in the near future? I'm confused about the relevance of the year.

Reply:
In the future, CPUs will be released which support the AH bit. So for each year that passes, the percentage of users that have a CPU with support for it will increase.

Reply:
Maybe I should clarify that "added in 2020" means added to AArch64, not to Dolphin.

Reply:
Reworded.

Reply:
Ok, so it's a hardware limitation, not of drivers or AArch itself. Got it